src/components/footer/Footer.jsx

```diff
@@ -1,57 +1,55 @@
-import React from 'react'
-import "./Footer.css"
+import React from "react";
+import "./footer.css";
 
 const Footer = () => {
   return (
     <footer className="footer">
-    <div className="footer-container">
-      <div className="footer__item">
-        <h4>Account</h4>
-        <ul className="footer__list">
-          <li className="footer__list-item">
-            <a href="#">My Account</a>
-          </li>
-          <li className="footer__list-item">
-            <a href="#">Wishlist</a>
-          </li>
-          <li className="footer__list-item">
-            <a href="#">Track Order</a>
-          </li>
-        </ul>
+      <div className="footer-container">
+        <div className="footer__item">
+          <h4>Account</h4>
+          <ul className="footer__list">
+            <li className="footer__list-item">
+              <a href="#">My Account</a>
+            </li>
+            <li className="footer__list-item">
+              <a href="#">Wishlist</a>
+            </li>
+            <li className="footer__list-item">
+              <a href="#">Track Order</a>
+            </li>
+          </ul>
+        </div>
+        <div className="footer__item">
+          <h4>Help</h4>
+          <ul className="footer__list">
+            <li className="footer__list-item">
+              <a href="#">FAQ</a>
+            </li>
+            <li className="footer__list-item">
+              <a href="#">Privacy Policy</a>
+            </li>
+          </ul>
+        </div>
```

Comment:
Suggestion: Here the structure of the code is similar so what you can do is put the link and corresponding text in an array of objects and map on that array.

Reply:
yes, I'll make these changes in future :)